src/components/MainNavbar.js

```diff
@@ -1,29 +1,24 @@
 import React from 'react';
 
-import { Nav, Navbar, NavbarBrand, NavLink } from 'react-bootstrap';
-import { LinkContainer } from 'react-router-bootstrap';
+import { Nav, Navbar } from 'react-bootstrap';
+import logo from '../assets/logo.png';
 
 class MainNavbar extends React.Component {
   render() {
     return (
-      <Navbar bg="dark" variant="dark" expand="lg">
-        <LinkContainer to="/">
-          <NavbarBrand Link>Build for Black Lives</NavbarBrand>
-        </LinkContainer>
-
-        <Nav style={{ marginLeft: 'auto' }}>
-          <LinkContainer to="/about">
-            <NavLink className="header-text">About</NavLink>
-          </LinkContainer>
-
-          <LinkContainer to="/projects">
-            <NavLink className="header-text">Open Projects</NavLink>
-          </LinkContainer>
-
-          <LinkContainer to="/project-request">
-            <NavLink className="header-text">Project Request</NavLink>
-          </LinkContainer>
-        </Nav>
+      <Navbar variant="dark"  expand="lg" sticky="top">
+        <Navbar.Brand href="/">
```

Comment:
Could you import NavbarBrand from react-bootstrap at the top so we can use that syntax instead of Navbar.Brand? (Repeat for the rest of the tags.

Reply:
@liu-caro I updated the `NavbarBrand` references but am getting errors when I try to switch from `Navbar.Toggle` to `NavbarToggle` or `Navbar.Collapse` to `NavbarCollapse`. I'm not super familiar with React, but the documentation I've found online is suggesting that they only exist as subclass versions. Is that okay?

Reply:
Yeah that's fine! Thanks for looking into it (: